SwiftPlate.xcodeproj/xcuserdata/Arthur.xcuserdatad/xcschemes/SwiftPlate.xcscheme

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Don't commit this. This is your private scheme. Come to think of it, SwiftPlate should include a shared scheme and this should be included in `.gitignore` 💡